projects/storefrontstyles/scss/components/pwa/add-to-home-screen-banner/_add-to-home-screen-banner.scss

```diff
@@ -0,0 +1,36 @@
+%cx-add-to-home-screen-banner {
```

Comment:
The build is failing because you should add this at the start of the file: `@import '../../../theme';`